presto-main/src/test/java/com/facebook/presto/operator/scalar/FunctionAssertions.java

```diff
@@ -167,17 +166,17 @@
 
     private static final Page ZERO_CHANNEL_PAGE = new Page(1);
 
-    private static final Map<Integer, Type> INPUT_TYPES = ImmutableMap.<Integer, Type>builder()
-            .put(0, BIGINT)
-            .put(1, VARCHAR)
-            .put(2, DOUBLE)
-            .put(3, BOOLEAN)
-            .put(4, BIGINT)
-            .put(5, VARCHAR)
-            .put(6, VARCHAR)
-            .put(7, TIMESTAMP_WITH_TIME_ZONE)
-            .put(8, VARBINARY)
-            .put(9, INTEGER)
+    private static final Map<Symbol, Type> INPUT_TYPES = ImmutableMap.<Symbol, Type>builder()
```

Comment:
Maybe we should change the `SYMBOL_TYPES` below to use this? This block looks the same to the part initializing `SYMBOL_TYPES`.